assets/runtime/config/gitlabhq/gitlab.yml

```diff
@@ -568,6 +568,19 @@ production: &base
     #   enabled: true
     #   host: localhost
     #   port: 3808
+    ## Monitoring
+  # Built in monitoring settings
+  monitoring:
+    # Time between sampling of unicorn socket metrics, in seconds
+    # unicorn_sampler_interval: 10
+    # IP whitelist to access monitoring endpoints
+    ip_whitelist:
+      - 127.0.0.0/8
```

Comment:
I think the default value should be here `0.0.0.0/8`. So that we can get access from outside.